The current driver is not setting the dev field in the private data structure,
which can lead to an OOPS if the driver tries to report an error.

Signed-off-by: Ben Dooks <ben-linux@fluff.org>

--- linux-2.6.20/drivers/video/s3c2410fb.c	2007-02-04 18:44:54.000000000 +0000
+++ linux-2.6.20-fbfix1/drivers/video/s3c2410fb.c	2007-02-15 22:42:51.000000000 +0000
@@ -791,6 +791,8 @@ static int __init s3c2410fb_probe(struct
 
 	info = fbinfo->par;
 	info->fb = fbinfo;
+	info->dev = &pdev->dev;
+
 	platform_set_drvdata(pdev, fbinfo);
 
 	dprintk("devinit\n");


